docs/workspaces/index.md

```diff
@@ -274,3 +274,19 @@ Instructors are strongly encouraged to allow students to get exposed to the Prai
 
 - Expectations about workspace start times. In the worst case, a workspace may take several minutes as new machines are started and as images are pulled. This may be especially important in timed assessments, where the workspace starting time may affect the student's ability to complete their work.
+
+## Troubleshooting workspace loading
+
+A variety of issues can prevent a workspace from loading, including antivirus software, browser extensions, and network issues. If a workspace fails to load, the following steps may help you resolve the issue or provide more information to the PrairieLearn team:
+
+- **Wait**: Wait for the loading progress indicator to reach 100%. This may take a few minutes if the workspace image is being pulled to a specific host for the first time. Even once a workspace's state changes to running, slow networks may result in a blank page for some time as the workspace's client-side assets load.
+- **Reboot:** If the workspace still hasn't loaded after a few minutes, try rebooting the workspace by clicking the "Reboot" button in the top nav bar. As above, give the workspace several minutes to load.
+- **Try another browser**: Some browser extensions may interfere with workspace loading. If you have another browser installed, try loading the workspace in that browser. Also consider trying a private/incognito window, which will disable most extensions.
+- **Check the browser developer tools console tab**: Open the browser's developer tools and navigate to the "Console" tab. This may contain error messages that can help diagnose the issue.
+  - On Chrome, you can open the developer tools by opening the Chrome menu in the top right corner, selecting "More tools", and then "Developer tools".
+  - On Safari, you have to turn on the developer tools first. In the menu bar, click "Safari" -> "Settings" -> "Advanced" and check "Show features for web developers". You can then open the developer tools by selecting "Develop" -> "Show Web Inspector".
+  - On Firefox, you can open the developer tools by opening the Firefox menu in the top right corner, selecting "More tools", and then "Web Developer Tools".
+- **Check the browser developer tools network tab**: Using the steps above, open the developer tools and navigate to the "Network" tab. This tab will show all network requests made by the browser, including those made by the workspace. Look for requests that are taking a long time or failing. You may need to reload the page to see all requests. If you want to provide the PrairieLearn team with more information, you can save the network log to a HAR file and send it. _Note: The HAR file may contain sensitive information, so be careful when sharing it. Only share it with individuals that you trust._
+  - On Chrome, click the downward-facing arrow in the top left corner (it has a tooltip with the text "Export HAR..." when you hover over it) and save the HAR file.
```

Comment:
The button for me is on the top menu but not on the left (checked both Chrome and Edge).

```suggestion
  - On Chrome or Edge, click the downward-facing arrow in the top menu (it has a tooltip with the text "Export HAR..." when you hover over it) and save the HAR file.
```

Reply:
I'd like to avoid the term "top menu" since that could mean something different on macOS. How's this instead?

```suggestion
  - On Chrome or Edge, click the downward-facing arrow on the Network tab menu bar (it has a tooltip with the text "Export HAR..." when you hover over it) and save the HAR file.
```

Reply:
Sure, that's good.